charts/px-central/templates/px-backup/pxcentral-backup.yaml

```diff
@@ -28,7 +28,7 @@ rules:
     resources: ["jobs"]
     verbs: ["get", "list"]
   - apiGroups: [""]
-    resources: ["namespaces"]
+    resources: ["namespaces", "services"]
```

Comment:
referring to similar implementation in line 100 to 108, can we add this in a separate block even when they share similar permissions?
@prashanthpx Need your input here